File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x<0:
         return False
        m = str(x)
        hi = ""
        for i in range(len(m)-1,-1,-1):
         hi = hi + m[i]
         
        k = int(hi)
        if k == x:
         return True
        else:
         return False

File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x<0){
            return false;
        }
        
        long long sum=0;
        int rem;
       int temp =x;
        while(temp!=0)
        {
            rem = temp%10;
            temp=temp/10;
            sum =sum *10 +rem;
        }
        if (sum == x)
{
    return true;
}
else{
    return false;
}

    }
};